Building Docker backend image throws error

```
#13 [builder 2/7] RUN --mount=type=cache,target=/root/.cache     --mount=type=bind,source=uv.lock,target=uv.lock     --mount=type=bind,source=pyproject.toml,target=pyproject.toml     uv sync         --locked         --no-dev         --no-group test         --group container         --no-install-project
#13 0.106 + uv sync --locked --no-dev --no-group test --group container --no-install-project
#13 0.264 Using CPython 3.12.13 interpreter at: /usr/local/bin/python3.12
#13 0.265 error: The requested interpreter resolved to Python 3.12.13, which is incompatible with the project's Python requirement: `>=3.13` (from `project.requires-python`)
#13 ERROR: process "sh -exc uv sync         --locked         --no-dev         --no-group test         --group container         --no-install-project" did not complete successfully: exit code: 2
------
 > [builder 2/7] RUN --mount=type=cache,target=/root/.cache     --mount=type=bind,source=uv.lock,target=uv.lock     --mount=type=bind,source=pyproject.toml,target=pyproject.toml     uv sync         --locked         --no-dev         --no-group test         --group container         --no-install-project:
0.106 + uv sync --locked --no-dev --no-group test --group container --no-install-project
0.264 Using CPython 3.12.13 interpreter at: /usr/local/bin/python3.12
0.265 error: The requested interpreter resolved to Python 3.12.13, which is incompatible with the project's Python requirement: `>=3.13` (from `project.requires-python`)
------
Dockerfile:7
--------------------
   6 |     # We expect pyproject.toml to have an entry cointainer inside [dependency-groups]
   7 | >>> RUN --mount=type=cache,target=/root/.cache \
   8 | >>>     --mount=type=bind,source=uv.lock,target=uv.lock \
   9 | >>>     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
  10 | >>>     uv sync \
  11 | >>>         --locked \
  12 | >>>         --no-dev \
  13 | >>>         --no-group test \
  14 | >>>         --group container \
  15 | >>>         --no-install-project
  16 |
--------------------
ERROR: failed to build: failed to solve: process "sh -exc uv sync         --locked         --no-dev         --no-group test         --group container         --no-install-project" did not complete successfully: exit code: 2

View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/m47e8peb7lw4d3kepinn09d02
make: *** [Makefile:166: build-image] Error 1
```

--- a/templates/ci/gh_project/{{ cookiecutter.__folder_name }}/workflows/backend.yml
+++ b/templates/ci/gh_project/{{ cookiecutter.__folder_name }}/workflows/backend.yml
@@ -65,7 +65,7 @@ jobs:
       image-name-suffix: {{ "${{ inputs.image-name-suffix }}" }}
       working-directory: {{ "${{ inputs.working-directory }}" }}
       build-args: |
-        PLONE_VERSION={{ "${{ inputs.plone-version }}" }}
+        PYTHON_VERSION={{ "${{ inputs.python-version }}" }}
       push: {{ "${{ github.event_name != 'pull_request' }}" }}
     secrets:
       username: {{ "${{ github.actor }}" }}

--- a/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Dockerfile
+++ b/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Dockerfile
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
-ARG PYTHON_VERSION=3.13
+ARG PYTHON_VERSION=undefined
 FROM plone/server-builder:uv-${PYTHON_VERSION} AS builder
 
 # Install project dependencies based on uv.lock and pyproject.toml

--- a/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Dockerfile
+++ b/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Dockerfile
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
-ARG PYTHON_VERSION=3.12
+ARG PYTHON_VERSION=3.13
 FROM plone/server-builder:uv-${PYTHON_VERSION} AS builder
 
 # Install project dependencies based on uv.lock and pyproject.toml

--- a/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
+++ b/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/Makefile
@@ -15,9 +15,6 @@ GREEN=`tput setaf 2`
 RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
-IMAGE_NAME_PREFIX={{ cookiecutter.__container_image_prefix }}
-IMAGE_TAG=latest
-
 # Python checks
 UV?=uv
 
@@ -33,6 +30,7 @@ IMAGE_NAME=$(IMAGE_NAME_PREFIX)-backend:$(IMAGE_TAG)
 
 PLONE_SITE_ID=Plone
 BACKEND_FOLDER=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+PYTHON_VERSION := $(shell echo '$(REPOSITORY_SETTINGS)' | jq -r '.backend.python_version')
 PLONE_VERSION := $(shell echo '$(REPOSITORY_SETTINGS)' | jq -r '.backend.base_package_version')
 EXAMPLE_CONTENT_FOLDER := $(shell echo '$(REPOSITORY_SETTINGS)' | jq -r '.backend.code_path')/setuphandlers/examplecontent
 PACKAGE_NAME := $(shell echo '$(REPOSITORY_SETTINGS)' | jq -r '.backend.name')
@@ -56,6 +54,7 @@ help: ## This help message
 .PHONY: debug-settings
 debug-settings:  ## Debug settings
 	@echo "Virtual Environment: $(VENV_FOLDER)"
+	@echo "PYTHON_VERSION: $(PYTHON_VERSION)"
 	@echo "PLONE_VERSION: $(PLONE_VERSION)"
 	@echo "PACKAGE_NAME: $(PACKAGE_NAME)"
 	@echo "EXAMPLE_CONTENT_FOLDER: $(EXAMPLE_CONTENT_FOLDER)"
@@ -164,7 +163,7 @@ test-coverage: $(VENV_FOLDER) ## run tests with coverage
 .PHONY: build-image
 build-image:  ## Build Container Images
 	@echo "$(GREEN)==> Building image $(IMAGE_NAME) $(RESET)"
-	@docker build . -t $(IMAGE_NAME) --progress plain -f Dockerfile
+	@docker build . -t $(IMAGE_NAME) --build-arg PYTHON_VERSION=$(PYTHON_VERSION) --progress plain -f Dockerfile
 
 ############################################
 # Acceptance Tests

--- a/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/pyproject.toml
+++ b/templates/sub/project_settings/{{ cookiecutter.__folder_name }}/backend/pyproject.toml
@@ -26,7 +26,6 @@ classifiers = [
     {%- if cookiecutter.__feature_distribution == '1' %}
     "Framework :: Plone :: Distribution",
     {%- endif %}
-    "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
     "Operating System :: OS Independent",
     "Programming Language :: Python",
     {%- for version in cookiecutter.__supported_versions_python %}
@@ -47,7 +46,7 @@ test = [
     "plone.restapi[test]",
     "pytest",
     "pytest-cov",
-    "pytest-plone>=1.0.0a2",
+    "pytest-plone>=1.0.0a3",
 ]
 
 container = [
@@ -189,7 +188,6 @@ ignore = [
     ".editorconfig",
     ".flake8",
     "dependabot.yml",
-    "mx.ini",
 ]
 
 [tool.pytest.ini_options]

--- a/tests/templates/sub/project_settings/test_sub_project_settings_variables.py
+++ b/tests/templates/sub/project_settings/test_sub_project_settings_variables.py
@@ -1,5 +1,9 @@
 ALLOWED_MISSING = []
-ALLOWED_NOT_USED = ["__generator_sha", "__version_plone_volto"]
+ALLOWED_NOT_USED = [
+    "__container_image_prefix",
+    "__generator_sha",
+    "__version_plone_volto",
+]
 
 
 def test_no_missing_variables(variables_missing):

